metadata.json

```diff
@@ -1,23 +1,79 @@
 {
-  "name": "mrpuppet-tagmail",
-  "version": "0.1.3",
-  "author": "mrpuppet",
-  "summary": "A report processor plugin for re-enabling the tagmail feature in PE 3.7 onwards. Use this to have Puppet send emails based on tags",
+  "name": "puppetlabs-tagmail",
+  "version": "1.0.0",
+  "author": "puppetlabs",
+  "summary": "This module provides a report processor that sends events to email addresses based on tagging of resources",
   "license": "Apache-2.0",
+  "source": "git://github.com/puppetlabs/puppetlabs-tagmail.git",
+  "project_page": "http://links.puppetlabs.com/tagmail-module",
+  "issues_url": "https://github.com/puppetlabs/puppetlabs-tagmail/issues",
   "operatingsystem_support": [
     {
-      "operatingsystem":"RedHat",
-      "operatingsystemrelease":[ "5.0", "6.0" ]
-    },
+    "operatingsystem": "RHEL",
+    "operatingsystemrelease": [
+      "5",
+      "6",
+      "7"
+    ]
+  },
+  {
+    "operatingsystem": "CentOS",
+    "operatingsystemrelease": [
+      "5",
+      "6",
+      "7"
+    ]
+  },
+  {
+    "operatingsystem": "Ubuntu",
+    "operatingsystemrelease": [
+      "10.04",
+      "12.04",
+      "14.04"
+    ]
+  },
+  {
+    "operatingsystem": "Debian",
+    "operatingsystemrelease": [
+      "6",
+      "7"
+    ]
+  },
+  {
+    "operatingsystem": "OracleLinux",
+    "operatingsystemrelease": [
+      "5",
+      "6",
+      "7"
+    ]
+  },
+  {
+    "operatingsystem": "Scientific",
+    "operatingsystemrelease": [
+      "5",
+      "6"
+    ]
+  },
+  {
+    "operatingsystem": "SLES",
+    "operatingsystemrelease": [
+      "11 SP1",
+      "12"
+    ]
+  }
+  ],
+  "requirements": [
     {
-      "operatingsystem": "Ubuntu",
-      "operatingsystemrelease": [
-        "12.04", "10.04"
-        ]
-    }
+    "name": "pe",
+    "version_requirement": ">= 3.7.0"
+  },
+  {
+    "name": "puppet",
+    "version_requirement": ">= 3.7.0"
+  }
   ],
-  "source": "https://github.com/anokun7/tagmail",
-  "project_page": "https://github.com/anokun7/tagmail",
-  "issues_url": "https://github.com/anokun7/tagmail/issues"
-}
+  "description": "This module provides a report processor that sends events to email addresses based on tagging of resources",
```

Comment:
This seems out of order. If you want to get the correct ordering, it's easiest to do a `puppet module build` and then `cp pkg/<tab>/metadata.json metadata.json` to let PMT reformat the json.